### PR TITLE
vim-patch:7.4.604

### PR DESCRIPTION
**Pull request by Shougo**
Problem:    Running tests changes viminfo.
Solution:   Disable viminfo.

https://github.com/vim/vim/commit/6bf7c523ad54f625547ef2cc6009c28c0dd337ed

### CODE REVIEW

File: src/nvim/testdir/test_breakindent.in [hunk 1]
```diff
@@ -101,7 +101,7 @@ fygjyl:let line2 = @0
 :$put =line2
 :"
 :let g:test="Test 14: breakindent + visual blockwise delete #1"
-:set all& breakindent
+:set all& breakindent viminfo+=nviminfo
```
Comment (ZyX-I) [inline src/nvim/testdir/test_breakindent.in]: I would suggest to use `shada+=nX-test-breakindent.shada`. `&viminfo` is now a deprecated option, ShaDa uses `.shada` extension and it is preferred to have at least test-file-local names of the temporary files.

Comment (Shougo) [inline src/nvim/testdir/test_breakindent.in]: Thank you.  Fixed.

**Comment by ZyX-I** [comment]
LGTM

**Comment by Shougo** [comment]
Thanks.

**Pull request merged**